goreleaser.yml

```diff
@@ -0,0 +1,11 @@
+builds:
+- skip: true
+
+release:
+  extra_files: 
+    - glob: ./docs.tar.gz
```

Comment:
I think this will upload the docs as that was being done previously.
We need to test that